### PR TITLE
Troubleshoot hotspot issues: add hotspot scenarios

### PR DESCRIPTION
**Pull request by en-jin19**
### What is changed, added or deleted? (Required)
Add two hotspot scenarios in "troubleshoot-hot-spot-issues.md".

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.1 (TiDB 6.1 versions)
- [x] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by rleungx — APPROVED** [review]

File: troubleshoot-hot-spot-issues.md [hunk 1]
```diff
@@ -48,6 +48,8 @@ Value: null
 
 同时，TiDB 中 RowID 默认也按照自增的方式顺序递增，主键不为整数类型时，同样会遇到写入热点的问题。
 
+此外，如需了解新建表或分区的写入热点问题和只读场景下周期性读热点问题可以参考[使用表属性控制 Region 合并的使用场景](/table-attributes.md#使用场景)。
```
Comment (TomShawn) [inline troubleshoot-hot-spot-issues.md:51]: 「## 常见热点场景」下都是陈述热点问题的，而后面都是说解决热点的方法。所以这一句放后面会不会更好呢？

Comment (en-jin19) [inline troubleshoot-hot-spot-issues.md:51]:
```suggestion
此外，当写入或读取数据存在热点时，即出现新建表或分区的写入热点问题和只读场景下周期性读热点问题时，你可以使用表属性控制 Region 合并。具体的热点场景描述和解决方法可以查看[使用表属性控制 Region 合并的使用场景](/table-attributes.md#使用场景)。

```

Comment (en-jin19) [reply troubleshoot-hot-spot-issues.md]: > 「## 常见热点场景」下都是陈述热点问题的，而后面都是说解决热点的方法。所以这一句放后面并加入更多的场景说明，会不会更好呢？

[使用表属性控制 Region 合并的使用场景](/table-attributes.md#使用场景) 链接的内容带有对具体场景的描述和热点处理方法。如果要挪动内容的话可能更需要把 table-attributes.md 下半章节的内容都挪动到本文档里并插入各个位置，而这样挪动可能不方便后续维护。如果之前的描述不太清晰的话，改为修改后的内容如何呢？

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**